src/pages/app/p/[package].settings.vue

```
@@ -214,6 +311,110 @@
+    return
+  }
+
+  const buttons = channels.map((chann) => {
+    return {
+      text: chann.name,
+      handler: async () => {
+        let appError: any
+        if (type !== 'both') {
+          appError = await supabase.from('apps')
+            .update({ [`default_channel_${type}`]: chann.id })
+            .eq('app_id', appRef.value?.app_id ?? '').then(x => x.error)
```

ESLint

[error] Should have line breaks between items, in node CallExpression

(antfu/consistent-chaining)